_docs/8_BlockExplorerSquid/blockexplorer.subspace.network

```diff
@@ -22,4 +22,26 @@ server {
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection "upgrade";
     }
+
+    location /db-health {
+        proxy_buffering off;
+        proxy_pass http://127.0.0.1:8080/health;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header Host $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "upgrade";
+    }
+
+    location /processor-health {
+        proxy_buffering off;
+        proxy_pass http://127.0.0.1:7070/health;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header Host $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "upgrade";
```

Comment:
There is no need to upgrade the connection if it's not websockets, just creates overhead each time for nginx in establishing a tunnel between client and backend.

Reply:
good to know, thanks!